services/horizon/internal/actions_metrics.go

```diff
@@ -84,7 +89,7 @@ func (action *MetricsAction) LoadSnapshot() {
 			values["1m.rate"] = t.Rate1()
 			values["5m.rate"] = t.Rate5()
 			values["15m.rate"] = t.Rate15()
-			values["mean.rate"] = t.RateMean()
+			values["mean"] = t.RateMean()
```

Comment:
this is a still a rate. why remove the `.rate` part from the name?

Reply:
it is the metrics exporter convention